src/tests/unit/sources.rs

```diff
@@ -34,9 +30,9 @@ pub async fn add_source() {
 }
 
 #[actix_rt::test]
-pub async fn add_source_with_invalid_name() {
+pub async fn when_add_source_with_invalid_name_expect_error() {

```

Comment:
Should be be expecting a `ResourecError::InvalidInput` error here? when_add_source_with_invalid_name_expect_invalid_input?